src/topology/uniform_space/completion.lean

```diff
@@ -407,6 +407,10 @@ lemma dense_inducing_coe : dense_inducing (coe : α → completion α) :=
 { dense := dense_range_coe,
   ..(uniform_inducing_coe α).inducing }
 
+/-- The uniform bijection between a complete space and its uniform completion. -/
+def uniform_completion.complete_equiv_self [complete_space α] [separated_space α]:
+completion α ≃ᵤ α := abstract_completion.complete_equiv_self completion.cpkg
```

Comment:
```suggestion
def uniform_completion.complete_equiv_self [complete_space α] [separated_space α] :
  completion α ≃ᵤ α := abstract_completion.complete_equiv_self completion.cpkg
```

Reply:
Thanks! Somehow I always get the formatting wrong. I'll push the changes in the next commit.